slider/beatmap.py

```diff
@@ -1910,12 +1910,8 @@ def hit_object_difficulty(self,
             Array of difficulties as ``float64``. Speed in the first column,
             aim in the second.
         """
-        cs = self.cs()
-        # NOTE: This is different than normal conversion
-        if hard_rock:
-            cs *= 1.3
-        elif easy:
-            cs /= 2
+        cs = self.cs(easy=easy,
```

Comment:
small style nit, but can you put the two arguments on a single line here and in the second call site below?